### PR TITLE
feat(constants): more constants

### PR DESCRIPTION
**Pull request by letypequividelespoubelles**

### CODE REVIEW

**Review by FlorianHuc — APPROVED** [review]
Try to avoid changing formating in such request to ease spot the changes.

File: constants/constants.lisp [hunk 1]
```diff
@@ -5,296 +5,297 @@
...
-  LINEA_SEPOLIA_CHAIN_ID                 0xe705
-  LINEA_BASE_FEE                         7
-  LINEA_DIFFICULTY                       2
+  LINEA_BLOCK_GAS_LIMIT                     61000000
+  LINEA_CHAIN_ID                            0xe708
+  LINEA_GOERLI_CHAIN_ID                     0xe704
+  LINEA_SEPOLIA_CHAIN_ID                    0xe705
+  LINEA_BASE_FEE                            7
+  LINEA_DIFFICULTY                          2
+  LINEA_MAX_NUMBER_OF_TRANSACTIONS_IN_BATCH 200
```
Comment (OlivierBBB) [inline constants/constants.lisp:224]: Is that the one new constant ?

**Review by OlivierBBB — APPROVED** [review]
LGTM

File: constants/constants.lisp (hunk 1 above)
Comment (letypequividelespoubelles) [reply constants/constants.lisp:224]: Yes

**Comment by letypequividelespoubelles** [comment]
> Try to avoid changing formating in such request to ease spot the changes.

I do agree , but we're using autoformating on save. And we need one common formatting

**Comment by lorenzogentile404** [comment]
@letypequividelespoubelles @OlivierBBB suggested to disable the formatting tool until we get a new one that reflects the emerging structures, that is what I am also doing. I am fine with both, as long as we all agree on something.

**Comment by letypequividelespoubelles** [comment]
I only use it for file that has be written with this formatting tool and disabled it for file with other formatting. ButvI can disabled it all the time ... But then we really need a common formatting tool.

**Pull request merged**